api/src/main/java/org/openmrs/module/xdssender/api/xds/XdsRetriever.java

```diff
@@ -27,30 +32,45 @@
 	@Autowired
 	private XdsSenderConfig config;
 	
-	public CloseableHttpResponse sendRetrieveCCD(String patientEcid) throws XDSException, IOException,
+	public HttpResponse sendRetrieveCCD(String patientEcid) throws XDSException, IOException,
```

Comment:
Should not the name of method  by typed  in the camelCase notation?

Reply:
It's not a method name, but the return type. Additionally those changes are already on master, and are shown here, probably because sth strange happend during merge.

Reply:
Ok, leave it like it is.